go/flags/endtoend/flags_test.go

```diff
@@ -67,6 +75,7 @@ func TestHelpOutput(t *testing.T) {
 			cmd := exec.Command(binary, args...)
 			output := bytes.Buffer{}
 			cmd.Stderr = &output
+			cmd.Stdout = &output
```

Comment:
Why did we need to add this?

Reply:
`vtctldclient` prints the help on Stdout and not on Stderr. Weird.

Reply:
Then I think we should fix that?

Reply:
I don't know how to. I checked and the help is coming from `cobra`. Is there some way to redirect it from Stdout to Stderr?